### PR TITLE
html-forms - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-03-29 at 10 43 44 AM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<html>
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <div>
+    <label for="email-address">Email</label>
```
Comment (shawnkost) [inline html-forms/index.html]: You're missing a `form` element! Everything from line 10 down should be wrapped inside a `form` element

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested here Bryan!

File: html-forms/index.html (hunk 1 above)
Comment (bbrown949) [reply html-forms/index.html]: fixed!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**